tendenci/apps/forms_builder/forms/models.py

```diff
@@ -234,6 +236,8 @@ class Field(OrderingBaseModel):
         help_text=_("Comma separated options where applicable"))
     default = models.CharField(_("Default"), max_length=1000, blank=True,
         help_text=_("Default value of the field"))
+    summary_position = models.CharField(_("Summary Position"), max_length=6, blank=True,
+        help_text=_("Position in the onle line form entry summary. Row, Position or just Position."))
```

Comment:
1) "onle line" or "one line"?

2) Can you please add some examples here so that users don't have to guess what can be entered.

3) The value of the `summary_position` field (entered by users) needs to be validated. This can be done by either adding a [validator](https://docs.djangoproject.com/en/3.2/ref/forms/validation/#using-validators) for the field `summary_position`, or adding a method `clean_summary_position` for the class `FormForField` in [forms.py](https://github.com/tendenci/tendenci/blob/master/tendenci/apps/forms_builder/forms/forms.py#L467) to throw [ValidationError](https://docs.djangoproject.com/en/3.2/ref/forms/validation/) if there is a problem with the field.

Reply:
Thanks! All good ideas. Will do.

Reply:
Hi Bernd,

Checking in to see what the outcome was of this request. I look forward to hearing from you!

Kind regards,
Nicole, Team Tendenci

Reply:
> Hi Bernd,
> 
> Checking in to see what the outcome was of this request. I look forward to hearing from you!
> 
> Kind regards, Nicole, Team Tendenci

Dropped the ball on this. Good reminder. Will leave the page open to take a look when I can squeeze the time in again. The validator is the thing that will take a moment to write, but is well worth doing indeed.

Reply:
Just fixed these three, but got the validator wrong. I just realised I have to write one after all ;-). And up the help a little.

Reply:
And now I fixed the validator. It provides rather comprehensive feedback, too.